src/ipa/rpi/controller/rpi/sync.cpp

```diff
@@ -119,100 +139,175 @@ void Sync::initialise()
 	socket_ = -1;
 }
 
+/* No idea what this does*/
```

Comment:
Probably don't need this comment, I don't think it helps anyone!! But for reference, `switchMode` gets called when the camera is stopped and reprogrammed, maybe transitioning from a smaller "preview" mode to a full resolution "capture" mode.